bluepyopt/ephys/morphologies.py

```diff
@@ -94,7 +94,7 @@ def instantiate(self, sim=None, icell=None):
         # TODO this is to get rid of stdout print of neuron
         # probably should be more intelligent here, and filter out the
         # lines we don't want
-        sim.neuron.h.hoc_stdout('/dev/null')
+        sim.neuron.h.hoc_stdout('null')
```

Comment:
Yeah, was thinking about that too. @Chris-crisur, would you mind trying that out (we don't have access to Windows machines here) ?